test/cypress/integration/test.acceptance.js

```diff
@@ -224,16 +224,5 @@ describe('Shepherd Acceptance Tests', () => {
         cy.get('.shepherd-step-element').should('have.length', 1);
       });
     });
-
-    describe('Cleaning up', () => {
```

Comment:
Shouldn't this test still work?

Reply:
https://github.com/shipshapecode/shepherd/pull/279#pullrequestreview-165925406